htdocs/web_portal/img/README.md

```diff
@@ -0,0 +1,22 @@
+| Icon image name | Link to source | License info |
```

Comment:
Can you add a column for the author information, this helps with compliance for some of the licenses (especially CC +Attribution).